assets/js/app.js

```diff
@@ -34,6 +35,24 @@ topbar.config({barColors: {0: "#29d"}, shadowColor: "rgba(0, 0, 0, .3)"})
 window.addEventListener("phx:page-loading-start", info => topbar.show())
 window.addEventListener("phx:page-loading-stop", info => topbar.hide())
 
+// initialize Vue Editor
+const el = document.getElementById("editor")
+
+if (el) {
+  const vueEditor = Editor.init(el, { content: JSON.parse(el.dataset.content)});
+
+  const articleForm = document.getElementById("article_form")
+  const articleSubmitButton = document.getElementById("article_submit")
+  const contentHiddenJsonField = document.getElementById("article_form_content_json")
+  const contentHiddenHtmlField = document.getElementById("article_form_content_html")
+
+  articleSubmitButton.addEventListener("click", () => {
+    contentHiddenJsonField.setAttribute("value", JSON.stringify(vueEditor.getJSON()))
+    contentHiddenHtmlField.setAttribute("value", vueEditor.getHTML())
+    articleForm.submit()
+  })
```

Comment:
Tried to do this with xhr/fetch api but got into trouble because of the redirect in elixir. So this is ugly but works